<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CardWindow</class>
 <widget class="QMainWindow" name="CardWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>488</width>
    <height>643</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="windowMessage">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Message</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="acceptDrops">
       <bool>false</bool>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="tabsClosable">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="studyTab">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <attribute name="title">
        <string>Study</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QComboBox" name="sortOption">
          <item>
           <property name="text">
            <string>Sort after date</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Sort after difficulty</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Sort after type</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Randomize</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QFrame" name="frame">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_16">
           <item>
            <widget class="QScrollArea" name="scrollArea">
             <property name="widgetResizable">
              <bool>true</bool>
             </property>
             <widget class="QWidget" name="scrollAreaWidgetContents_4">
              <property name="geometry">
               <rect>
                <x>0</x>
                <y>0</y>
                <width>426</width>
                <height>411</height>
               </rect>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_9">
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_14">
                 <item>
                  <layout class="QVBoxLayout" name="questionAreaStudy"/>
                 </item>
                 <item>
                  <widget class="Line" name="line_4">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <layout class="QVBoxLayout" name="answerAreaStudy"/>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="showAnswerButton">
             <property name="text">
              <string>Show answer</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_15">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_5">
               <item>
                <widget class="QPushButton" name="vhButton">
                 <property name="text">
                  <string>Very hard</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="hButton">
                 <property name="text">
                  <string>Hard</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="mButton">
                 <property name="text">
                  <string>Medium</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="eButton">
                 <property name="text">
                  <string>Easy</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="veButton">
                 <property name="text">
                  <string>Very easy</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="editTab">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <attribute name="title">
        <string>Edit</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QFrame" name="frame_2">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_17">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_9">
             <item>
              <widget class="QLabel" name="label_8">
               <property name="text">
                <string>Question Type</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="answerTypeEdit">
               <item>
                <property name="text">
                 <string>Text</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Image</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Image Text</string>
                </property>
               </item>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_7">
               <property name="text">
                <string>Answer type:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="questionTypeEdit">
               <item>
                <property name="text">
                 <string>Text</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Image</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Image Text</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QScrollArea" name="scrollArea_2">
             <property name="widgetResizable">
              <bool>true</bool>
             </property>
             <widget class="QWidget" name="scrollAreaWidgetContents_5">
              <property name="geometry">
               <rect>
                <x>0</x>
                <y>0</y>
                <width>426</width>
                <height>436</height>
               </rect>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_13">
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_18">
                 <item>
                  <layout class="QVBoxLayout" name="questionAreaEdit">
                   <item>
                    <widget class="QLabel" name="label">
                     <property name="text">
                      <string/>
                     </property>
                     <property name="pixmap">
                      <pixmap>../../Desktop/swe.jpg</pixmap>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <widget class="Line" name="line_5">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <layout class="QVBoxLayout" name="answerAreaEdit"/>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_19">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_7">
               <item>
                <widget class="QPushButton" name="saveChangesButton">
                 <property name="text">
                  <string>Save changes</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="deleteCardButton">
                 <property name="text">
                  <string>Delete card</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="newTab">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <attribute name="title">
        <string>Add new card</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QFrame" name="frame_3">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_20">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_10">
             <item>
              <widget class="QLabel" name="label_10">
               <property name="text">
                <string>Question Type</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="newQuestionType">
               <item>
                <property name="text">
                 <string>Text</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Image</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Image Text</string>
                </property>
               </item>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_9">
               <property name="text">
                <string>Answer type:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="newAnswerType">
               <item>
                <property name="text">
                 <string>Text</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Image</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Image Text</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QScrollArea" name="scrollArea_3">
             <property name="widgetResizable">
              <bool>true</bool>
             </property>
             <widget class="QWidget" name="scrollAreaWidgetContents_6">
              <property name="geometry">
               <rect>
                <x>0</x>
                <y>0</y>
                <width>426</width>
                <height>436</height>
               </rect>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_4">
               <item>
                <layout class="QVBoxLayout" name="questionAreaNew">
                 <item>
                  <widget class="QPushButton" name="newQuestionImageButton">
                   <property name="text">
                    <string>Add image</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="newQuestionImage">
                   <property name="text">
                    <string/>
                   </property>
                   <property name="alignment">
                    <set>Qt::AlignCenter</set>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPlainTextEdit" name="newQuestionText">
                   <property name="enabled">
                    <bool>true</bool>
                   </property>
                   <property name="plainText">
                    <string>Write question</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <widget class="Line" name="line_6">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                </widget>
               </item>
               <item>
                <layout class="QVBoxLayout" name="answerAreaNew">
                 <item>
                  <widget class="QPushButton" name="newAnswerImageButton">
                   <property name="text">
                    <string>Add image</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="newAnswerImage">
                   <property name="text">
                    <string/>
                   </property>
                   <property name="alignment">
                    <set>Qt::AlignCenter</set>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPlainTextEdit" name="newAnswerText">
                   <property name="plainText">
                    <string>Write answer</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_23">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_8">
               <item>
                <widget class="QPushButton" name="newAddButton">
                 <property name="text">
                  <string>Add new card</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="newClearButton">
                 <property name="text">
                  <string>Clear card</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>488</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
